hypertrace-graphql-metric-schema/src/main/java/org/hypertrace/graphql/metric/schema/BaselineMetricAggregation.java

```diff
@@ -0,0 +1,15 @@
+package org.hypertrace.graphql.metric.schema;
+
+import graphql.annotations.annotationTypes.GraphQLField;
+import graphql.annotations.annotationTypes.GraphQLName;
+
+@GraphQLName(BaselineMetricAggregation.BASELINE_METRIC_AGGREGATION_TYPE_NAME)
+public interface BaselineMetricAggregation extends MetricAggregation {
```

Comment:
Naming throughout - this name sounds like it's a baseline value, when in reality it's both an aggregation and it's baseline. Could we rename it `BaselinedMetricAggregation` or propose a new name?